bucket/robo3t.json

```diff
@@ -16,11 +16,15 @@
             "Robo 3T"
         ]
     ],
+    "checkver": {
+        "github": "https://github.com/Studio3T/robomongo",
+        "regex": "robo3t-(?<version>[\\d.]+)-windows-x86_64-(?<commitHash>[0-9a-f]+)"
+    },
     "autoupdate": {
         "architecture": {
             "64bit": {
-                "url": "https://download.studio3t.com/robomongo/windows/robo3t-$version-windows-x86_64-$matchRandom.zip",
-                "extract_dir": "robo3t-$version-windows-x86_64-$matchRandom"
+                "url": "https://github.com/Studio3T/robomongo/releases/download/v$version/robo3t-$version-windows-x86_64-$matchCommitHash.zip",
+                "extract_dir": "robo3t-$version-windows-x86_64-$matchCommitHash"
```

Comment:
Based on offline checks, it looks like Scoop prefers matchCommithash instead of matchCommitHash

Reply:
@igitur you will need to change extract_dir to robo3t-$version-windows-x86_64-$matchCommithash